agent/config/config.go

```diff
@@ -58,6 +58,14 @@ func Parse(data string, format string) (c Config, md mapstructure.Metadata, err
 	return c, md, nil
 }
 
+// CacheConfig Cache tunning configuration
+type CacheConfig struct {
+	// EntryFetchMaxBurst max burst size of RateLimit for a single cache entry
+	EntryFetchMaxBurst *int `json:"rate_limit_burst,omitempty" hcl:"rate_limit_burst" mapstructure:"rate_limit_burst"`
+	// EntryFetchRateLimit represents the max calls/sec for a single cache entry
+	EntryFetchRateLimit *float64 `json:"rate_limit_per_entry,omitempty" hcl:"rate_limit_per_entry" mapstructure:"rate_limit_per_entry"`
```

Comment:
```suggestion
	EntryFetchRateLimit *float64 `mapstructure:"entry_fetch_rate_limit"`
```

Reply:
fixed in https://github.com/hashicorp/consul/pull/8226/commits/1e0b1d2886f2108d18f4385d7ed1625ae6a946ec